### PR TITLE
[FIX] Retirement tests

### PR DESCRIPTION
**Pull request by lehinevych**
{{pytest: cfme/tests/cloud_infra_common/test_retirement.py cfme/tests/infrastructure/test_vm_retirement_rest.py -v --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 6c962afdbdf59f87a182fcd9009270638182e5f8:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ffff5803d74dac7ca96bc4a85b09a3a3a703203a:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by lehinevych** [comment]
failed on upstream because of https://github.com/ManageIQ/manageiq/issues/7561

**Comment by dajoRH** [comment]
Lint report for commit cec0af71fcd2d2d7fbfbe17751d654de77b6ce57:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/infrastructure/test_vm_retirement_rest.py [hunk 1]
```diff
@@ -48,17 +48,8 @@ def test_retire_vm_now(rest_api, vm, multiple):
...
-        # The retirement_state field appears after calling retire method
-        try:
-            if retire_vm.retirement_state == "retired":
-                return True
-        except:
-            pass
-        return False
-
-    wait_for(_finished, num_sec=600, delay=10, message="REST vm retire now")
+    wait_for(lambda: not rest_api.collections.vms.get(name=vm),
```
Comment (psav) [inline cfme/tests/infrastructure/test_vm_retirement_rest.py]: The previous check checked for a specific state, is this new method as specific?

Comment (lehinevych) [inline cfme/tests/infrastructure/test_vm_retirement_rest.py]: Now the retired vm is deleted, so you can't check specific state (unavailable via rest and ui)

**Comment by dajoRH** [comment]
Lint report for commit 4640af2f066d59f519bd12fe556120cb78171850:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 9ab3228c690bbd25c18b22950b6e9103f60a3507:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 06de5de0d1088c5e30b7fb892b218f238096121f:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request closed without merging**